lib/client/api.go

```diff
@@ -1933,6 +1933,11 @@ func (tc *TeleportClient) runShell(nodeClient *NodeClient, sessToJoin *session.S
 		return trace.Wrap(err)
 	}
 	if err = nodeSession.runShell(tc.OnShellCreated); err != nil {
+		originErr := trace.Unwrap(err)
+		exitErr, ok := originErr.(*ssh.ExitError)
+		if ok {
```

Comment:
```suggestion
		if exitErr, ok := trace.Unwrap(err).(*ssh.ExitError); ok {
```
might be more idiomatic.